react-graphql-apollo-tutorial.md

```diff
@@ -100,9 +98,9 @@ const client = new ApolloClient({
 });
 {{< /highlight >}}
 
-You may recall how you did the same for the previous application that has only used axios for plain HTTP requests. You configured axios once with the GraphQL API endpoint to default all requests to this URI and set up the authorization header as well. The same happened here, because it is sufficient to configure your client once for all the following GraphQL requests.
+You did the same for the previous application, using only axios for plain HTTP requests. You configured Axios once with the GraphQL API endpoint to default all requests to this URI, and set up the authorization header. The same happened here, because it's enough to configure your client once for all the following GraphQL requests.
```

Comment:
"... configured Axios once with the GraphQL API ..." to "... configured axios once with the GraphQL API ..."

Reply:
I made these changes, but I am curious: why is axios not capitalized? Isn't it the name of a library?

Reply:
Yes, that's true. I tried to reference libraries as their npm package names (e.g. graphql-js, apollo-client, react-apollo) so that people can look them up easily. That's why I would not capitalize axios. On the other hand, I know that I have used React Apollo or Apollo Client too. Then it would be okay again to capitalize it. Matter of taste in the end, so it wasn't too important :) I think I have written it axios without capitalizing it in other tutorials too.